framework_tvb/tvb/adapters/datatypes/db/connectivity.py

```diff
@@ -48,6 +48,9 @@ class ConnectivityIndex(DataType):
     tract_lengths_max = Column(Float)
     tract_lengths_mean = Column(Float)
 
+    cortical_mask = Column(Boolean)
+    hemispheres_mask = Column(Boolean)
```

Comment:
Maybe rename these into `has_cortical_mask` and `has_hemisphere_mask` to signal that is a boolean?

Reply:
Done.